port/common/omrmem32helpers.c

```diff
@@ -37,26 +37,14 @@ static void *reserveAndCommitRegion(struct OMRPortLibrary *portLibrary, uintptr_
 #define VMEM_MODE_COMMIT OMRPORT_VMEM_MEMORY_MODE_READ | OMRPORT_VMEM_MEMORY_MODE_WRITE | OMRPORT_VMEM_MEMORY_MODE_COMMIT
 #define VMEM_MODE_WITHOUT_COMMIT OMRPORT_VMEM_MEMORY_MODE_READ | OMRPORT_VMEM_MEMORY_MODE_WRITE
 
+#define MEM32_LIMIT 0XFFFFFFFF
```

Comment:
I understand you're just moving this macro, but it should have been an unsigned quanitity, perhaps with a cast:
```c
#define MEM32_LIMIT ((uintptr_t)0XFFFFFFFFU)
```